pkgs/by-name/co/coturn/package.nix

```diff
@@ -42,6 +42,11 @@ stdenv.mkDerivation rec {
     ./dont-call-setgroups-unconditionally.patch
   ];
 
+  postPatch = ''
+    # contains brocken symlinks
+    rm examples/etc/*.pem
```

Comment:
```suggestion
    rm -r examples
```

Reply:
also: "brocken". (Have you climbed it recently? ;))

Reply:
Unfortunately just removing the examples directory doesn't work. I've pushed another fix that disabled installing the examples via a configure flags.